torchvision/prototype/models/optical_flow/raft.py

```diff
@@ -23,16 +21,27 @@
 
 
 class Raft_Large_Weights(WeightsEnum):
-    pass
-    # C_T_V1 = Weights(
-    #     # Chairs + Things
-    #     url="",
-    #     transforms=RaftEval,
-    #     meta={
-    #         "recipe": "",
-    #         "epe": -1234,
-    #     },
-    # )
+    C_T_V1 = Weights(
+        # Chairs + Things, ported from original paper repo (raft-things.pth)
+        url="https://download.pytorch.org/models/raft_large_C_T_V1-22a6c225.pth",
+        transforms=RaftEval,
+        meta={
```

Comment:
Question: Can Raft handle images of fixed or variable size? If the size of input image is fixed, we should add add a resize in the RaftEval transforms and put the tuple size in the meta similar to:
https://github.com/pytorch/vision/blob/5dc1e20b88f92dda862a7aab522bfdac2e6bd3b8/torchvision/prototype/models/alexnet.py#L21

If the model handles variable size images, you don't do anything.

Reply:
Images just need to be a multiple of 8 (and big enough) but otherwise there is no constraint on the size